src/Goerli-DssSpellCollateral.sol

```diff
@@ -28,61 +60,252 @@ contract DssSpellCollateralAction {
+    // -- RWA008 end --
+
+    // -- RWA009 MIP21 components --
+    address constant RWA009                  = 0xd2B101854F64Df738bA601840279838568583F39;
+    address constant MCD_JOIN_RWA009_A       = 0x7122B934F02A15954282Ed41572Ada539864773a;
+    address constant RWA009_A_URN            = 0xD2C8588C72026171Ec3a17369ad0f0734E30915d;
+    address constant RWA009_A_JAR            = 0xa484C16D2Ca15706c4B875710d9e80b7F101572B;
+    // Goerli: CES Goerli Multisig / Mainnet: Genesis
+    address constant RWA009_A_OUTPUT_CONDUIT = 0x7a3D23Dc73F7ead55399597aAE6e525b3DF95A88;
+
+    // MIP21_LIQUIDATION_ORACLE params
+    uint256 constant RWA009_A_INITIAL_DC     = 100_000_000 * RAD;
+    uint256 constant RWA009_A_INITIAL_PRICE  = 100_000_000 * WAD;
+    uint48  constant RWA009_A_TAU            = 0;
+
+    uint256 constant RWA009_REG_CLASS_RWA    = 3;
+
+    string constant RWA009_DOC               = "QmZG31b6iLGGCLGD7ZUn8EDkE9kANPVMcHzEYkvyNWCZpG";
+    // -- RWA009 END --
+
+    function onboardRwa008(
+        ChainlogAbstract CHANGELOG,
+        IlkRegistryAbstract REGISTRY,
+        address MIP21_LIQUIDATION_ORACLE,
+        address MCD_VAT,
+        address MCD_JUG,
+        address MCD_SPOT,
+        address MCD_JOIN_DAI
+    ) internal {
+        // RWA008-A collateral deploy
+
+        bytes32 ilk      = "RWA008-A";
+        uint256 decimals = DSTokenAbstract(RWA008).decimals();
+
+        // Sanity checks
+        require(GemJoinAbstract(MCD_JOIN_RWA008_A).vat() == MCD_VAT,  "join-vat-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA008_A).ilk() == ilk,      "join-ilk-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA008_A).gem() == RWA008,   "join-gem-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA008_A).dec() == decimals, "join-dec-not-match");
+
+        require(RwaUrnLike(RWA008_A_URN).vat() == MCD_VAT,               "urn-vat-not-match");
+        require(RwaUrnLike(RWA008_A_URN).jug() == MCD_JUG,               "urn-jug-not-match");
+        require(RwaUrnLike(RWA008_A_URN).daiJoin() == MCD_JOIN_DAI,      "urn-daijoin-not-match");
+        require(RwaUrnLike(RWA008_A_URN).gemJoin() == MCD_JOIN_RWA008_A, "urn-gemjoin-not-match");
+
+        // Init the RwaLiquidationOracle
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(ilk, RWA008_A_INITIAL_PRICE, RWA008_DOC, RWA008_A_TAU);
+        (, address pip, , ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+
+        // Set price feed for RWA008
+        SpotAbstract(MCD_SPOT).file(ilk, "pip", pip);
+
+        // Init RWA008 in Vat
+        VatAbstract(MCD_VAT).init(ilk);
+        // Init RWA008 in Jug
+        JugAbstract(MCD_JUG).init(ilk);
+
+        // Allow RWA008 Join to modify Vat registry
+        VatAbstract(MCD_VAT).rely(MCD_JOIN_RWA008_A);
+
+        // Allow RwaLiquidationOracle to modify Vat registry
+        VatAbstract(MCD_VAT).rely(MIP21_LIQUIDATION_ORACLE);
```

Comment:
This permission is already set as is global for all the RWAs. I'd comment it with a message if we want to keep it just for documentation.

Reply:
Apparently it's not, at least for Goerli.
If I remove this from both onboarding functions, some tests fail.
I left this in one of them and commented out on the other.

Reply:
Hmm will look into it in more detail. Pretty weird, but I have missed it when we migrated from kovan to goerli.